Gemfile.lock

```diff
@@ -262,8 +260,8 @@ GEM
     responders (3.0.1)
       actionpack (>= 5.0)
       railties (>= 5.0)
-    rexml (3.2.4)
-    rubocop (1.8.1)
+    rexml (3.2.5)
+    rubocop (1.10.0)
```

Comment:
### [rubocop](https://rubocop.org/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.8.1 → 1.10.0

[_[change-log](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop-hq/rubocop/)_]





<details>
<summary>Commits</summary>

A change of **128** commits. See the full changes on [the compare page](https://github.com/rubocop/rubocop/compare/v1.8.1...v1.10.0).

These are the first 10 commits:
- (562682a) [Switch back the docs version to master](https://github.com/rubocop/rubocop/commit/562682a3db33cad154a2cb70f18aff9e482bd2cd)
- (1903522) [[Fix #9353] Add new `Lint/SymbolConversion` cop.](https://github.com/rubocop/rubocop/commit/1903522e40fee189f26490c971c4d28a30c51c05)
- (fe7419f) [Correct `Lint/SymbolConversion` offenses.](https://github.com/rubocop/rubocop/commit/fe7419f9f93675acf4b9463815b0b4ae688d0d2e)
- (02ac59e) [Add new `Lint/NumberedParameterAssignment` cop](https://github.com/rubocop/rubocop/commit/02ac59e55daa28fb63c85e19d4e91d0355c0c148)
- (3e97bcd) [Allow a `Config` object to be created without running valida…](https://github.com/rubocop/rubocop/commit/3e97bcdad0b81483fd53e548c0cd72c5c5af145c)
- (d4d1b3a) [Change ConfigLoader#added_features to be a Set.](https://github.com/rubocop/rubocop/commit/d4d1b3a927e79e45d6628bbf3f627048120ee4a2)
- (0bf1e53) [Fix errors being reported with `rubocop -V` with an invalid …](https://github.com/rubocop/rubocop/commit/0bf1e530011bee441fe34090d2e12122f23b4d76)
- (860e681) [[Fix #9366] Fix an incorrect auto-correct for `Style/SoleNes…](https://github.com/rubocop/rubocop/commit/860e6813da66cd1201901bb489eba8e5fec5e472)
- (3dc4e62) [Fix typo for AllowedIdentifiers example](https://github.com/rubocop/rubocop/commit/3dc4e6298e6caa96fcc11a6d2a614ce6a2d4c44d)
- (0a53922) [[Fix #9370] Fix an incorrect auto-correct for `Style/SoleNes…](https://github.com/rubocop/rubocop/commit/0a539222132ff230272dca3f1b2ddcfd0e5fc940)


</details>